modules/systemd/README.md

```diff
@@ -1,5 +1,14 @@
 # `systemd`
 
-The `systemd` module streamlines the management of systemd units during image building. Units are divided into `system` and `user` categories, with `system` units managed directly using `systemctl` and `user` units using `systemctl --global`. You can specify which units to enable/disable or unmask/mask under each category.
+The `systemd` module streamlines the inclusion & management of systemd units during image building. Units are divided into `system` and `user` categories, with `system` units managed directly using `systemctl` and `user` units using `systemctl --global`. You can specify which units to enable/disable or unmask/mask under each category.
```

Comment:
I'm not sure if "inclusion" is the right term for this. I don't think we need to highlight this feature at all. Just mention that `systemd` units can be placed in the `config/systemd/{system, user}` directories, or directly in their correct place in `/usr/lib/systemd/{system, user}`.

Reply:
I removed inclusion term.

Do you want me to modify to use `config/systemd/{system, user}` instead of:

`config/systemd/system`
`config/systemd/user`

I use right now?